include/evmc/mocked_host.hpp

```diff
@@ -156,6 +156,15 @@ class MockedHost : public Host
         return accounts.count(addr) != 0;
     }
 
+    /// Access an account (EIP-2929).
+    evmc_access_status access_account(const address&) noexcept override { return EVMC_COLD_ACCESS; }
```

Comment:
For later: we should implement some basic map of access to also return "warm" status. This will help with code coverage in evmone.